src/components/Footer/Footer.tsx

```diff
@@ -1,72 +1,71 @@
-import { Link } from 'react-router-dom';
+import { useState } from 'react';
 
 import logo from 'src/assets/csa-logos/cluster-logo.svg';
 import feedbackIcon from 'src/assets/feedback.svg';
-import { ROUTES } from 'src/routing/routes';
 
+import FeedbackModal from 'components/modals/FeedbackModal';
 import Button from 'components/ui/Button';
+import Container from 'components/ui/Container';
 
 import Contacts from './Contacts';
 import SocialList from './SocialList';
 
 import styles from './Footer.module.css';
 
 const Footer = () => {
-  const navItems = [
-    { route: ROUTES.HOME, text: 'Terms of service' },
-    { route: ROUTES.HOME, text: 'Privacy policy' },
-    { route: ROUTES.HOME, text: 'Cookie policy' },
-  ];
+  const [isShowModal, setIsShowModal] = useState<'feedback' | null>(null);
+
+  const toggleModal = (name: typeof isShowModal) => () => {
```

Comment:
why don't you just use a boolean in showModal logic? Also you don't need to create a func which returns another one

Reply:
toggleModal - bad name for a function 
 You can read a good article: https://jaketrent.com/post/naming-event-handlers-react/